scripts/build-swagger-codegen.sh

```diff
@@ -1,6 +1,5 @@
-#!/bin/bash
-
-SWAGGER_VERSION="2.3.1"
+SWAGGER_VERSION="2.2.3"
+API_LANGUAGE="python"
```

Comment:
Since `python` is hardcoded, we won't be able to generate a new version of the docker image for typescript.
`API_LANGUAGE` (and maybe `SWAGGER_VERSION`) shoud be passed as build-args (https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg)